src/prompt_constructor.py

```diff
@@ -123,8 +161,9 @@ def prompt_generate_custom_cuda_fewshot_and_template(ref_arch_src: str, shots: l
     example_tiled_matmul_new = read_file(
         os.path.join(REPO_TOP_PATH, "src/prompts/few_shot/model_new_ex_tiled_matmul.py")
     )
-    example_tiled_matmul_desc = "This given architecture is for a model with tiled matrix multiplication: "
-
+    example_tiled_matmul_desc = (
```

Comment:
can we split out the linting changes? or least seperate things out by adding ruff in CI?